examples/hb-subset.example.node.js

```diff
@@ -20,9 +21,9 @@ const writeFileAsync = require('util').promisify(fs.writeFile);
     /* Add your glyph indices here and subset */
     const input = exports.hb_subset_input_create_or_fail();
     const unicode_set = exports.hb_subset_input_unicode_set(input);
-    exports.hb_set_add(unicode_set, 'a'.charCodeAt(0));
-    exports.hb_set_add(unicode_set, 'b'.charCodeAt(0));
-    exports.hb_set_add(unicode_set, 'c'.charCodeAt(0));
+    for (const text of SUBSET_TEXT) {
+        exports.hb_set_add(unicode_set, text.charCodeAt(0));
```

Comment:
Though it is safe for this example (the text is all ASCII), `charCodeAt` gives UTF-16 code units (including surrogate pairs), you should use `codePointAt` instead.

Reply:
Thanks for the correction, let me fix it.